patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Retrospectives for continuous improvement
```

Comment:
I would include the cross-team bit here, as that is the key difference to a regular 1-team retro here.

Or maybe "Maintainer + Contributor Retrospectives" or similar.